e2e/config.yaml

```diff
@@ -3,3 +3,8 @@ flows:
   - screens/*
   - settings/*
   - transactions/*
+platform:
```

Comment:
Interesting, did that help with a particular failure? Or just kind makes everything faster?

Reply:
seems to just help generally. found it was an option and thought might as well try this. i think it may impact iOS more tbh